lib/runners/swift.js

```diff
@@ -52,7 +52,7 @@ module.exports.run = function run(opts, cb) {
         args.push('/workspace/setup.swift');
       }
       fs.writeFileSync('/workspace/solution.swift', opts.solution);
-      fs.writeFileSync('/workspace/main.swift', opts.fixture);
+      fs.writeFileSync('/workspace/main.swift', opts.fixture.replace(/^\s*XCTMain\(/m, '_XCTMain('));
```

Comment:
We may or may not want to support both with and without a leading `_`. We should probably support whitespace in between `n(` just in case.